pkg/kf/commands/apps/push_test.go

```diff
@@ -113,6 +111,12 @@ func TestPushCommand(t *testing.T) {
 		},
 	} {
 		t.Run(tn, func(t *testing.T) {
+
+			if tc.path != "" {
```

Comment:
Now the push command checks the `path` directory and requires that it exists.

The only issue with this is: testing for a non-existent path isn't possible. I'm open to ideas.